src/averageAges.js

```diff
@@ -20,6 +20,31 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const ages = [];
+  let averageAge = 0;
+  let eteration = 0;
+
+  century
+    ? people.map((human) => {
+      if (human.sex === 'm' && Math.ceil(human.died / 100) === century) {
```

Comment:
* Consider creating constants with meaningful names instead of using magic strings and numbers
* Consider creating a helper function to check the person's sex, for example isMan or isWoman
``` const isMan = (person) => person.sex === MAN ```  It would help to follow DRY principle